activerecord/CHANGELOG.md

```diff
@@ -16,7 +28,6 @@
 
     *Leonardo Luarte*
 
-
```

Comment:
No idea how, but this was removed from `main` so if you rebase this change should be cleared. And makes a much cleaner merge / revert / backport if necessary. :pray: